tests/scripts/kubeadm-install.sh

```diff
@@ -47,7 +47,7 @@ sudo apt-get update
 wait_for_dpkg_unlock
 sleep 5
 wait_for_dpkg_unlock
-
+sudo apt-get install -y kubernetes-cni="0.6.0-00"
```

Comment:
is this a new requirement? or does this help address flakiness we've seen?

Reply:
Hey @jbw976 , it addresses an issue brought up by a community member. All current integration tests are failing because a new requirement was introduced for running Kubernetes in Ubuntu. I addressed the issue in [2906](https://github.com/rook/rook/issues/2906). This is needed for integration tests to work.

Reply:
should this kubernetes-cni install version be generated similar to what we do with `KUBE_INSTALL_VERSION`? defining them all together at the top may make maintenance of this easier so that their versions are bumped together.

Do we understand why kubelet requires `0.6.0` and the default `0.7.5-00` was not OK? will this change in the future? do we know how the versioning works here?

Reply:
While doing some digging, this issue seems to have emerged around three days ago. 
[Issue #75683 on Kubernetes](https://github.com/kubernetes/kubernetes/issues/75683)

As one user mentions: 
```
Using kubernetes-cni 0.7.5 works if you're pulling down k8s 1.14. For previous 
versions, it appears you need to still pin the package to 0.6.0.
```

During my testing with Ubuntu 16.04 today with a variety of Kubernetes versions < 1.14 (all of our current integration tests are less than 1.14), the version had to be pinned to 0.6.0. Kubernetes 1.14 works just fine with the default version of 0.7.5. 

I'd like to use newer versions of Kubernetes in our integration tests in another PR. As this is really a "break/fix" PR, I think we should hard code it to get the engine running again. I'd be happy to put in some logic in the shell script to detect if a version less that 1.14 is used. 

What do you think we should do now?

Reply:
Thanks for the explanation @Defilan.  I agree that getting this hard coded fix in to unblock integration tests again is OK.  Then let's please open an issue tracking a more long term solution that uses 0.7.5 for k8s versions 1.14+, as [this comment](https://github.com/kubernetes/kubernetes/issues/75683#issuecomment-476649311) notes there is a CVE issue in 0.6.0:
```
As people are fixing this -- keep in mind that there appears to be an embargoed CVE on kubernetes-cni=0.6.0, so be wary of pinning your start up scripts to that version permanently.
```
In that new issue, let's also make a note to keep an eye on if they fix the packages upstream so that 0.7.5 can be used for k8s versions before 1.14 also.  It'd be good to no longer use 0.6.0 anywhere.

Reply:
@jbw976 @Defilan 1.13.5 onwards uses kubernetes-cni 0.7.5